src/ServerlessBuildPlugin.ts

```diff
@@ -239,12 +240,43 @@ export class ServerlessBuildPlugin {
     }
   }
 
+  async buildFunction (fnName, fnConfig) {
+
+    const runtime = (fnConfig.runtime && fnConfig.runtime) || this.serverless.config.service.runtime;
```

Comment:
Whats with `(fnConfig.runtime && fnConfig.runtime)`

Reply:
yeah oops, I'll fix that